repo/packages/M/marathon/17/resource.json

```diff
@@ -0,0 +1,14 @@
+{
+  "images" : {
+    "icon-small" : "https://downloads.mesosphere.com/marathon/assets/icon-service-marathon-small.png",
+    "icon-medium" : "https://downloads.mesosphere.com/marathon/assets/icon-service-marathon-medium.png",
+    "icon-large" : "https://downloads.mesosphere.com/marathon/assets/icon-service-marathon-large.png"
+  },
+  "assets" : {
+    "container" : {
+      "docker" : {
+        "image" : "mesosphere/marathon:v1.4.7"
```

Comment:
`1.4.9`?

Reply:
strange... I changed these... let me fix it